android/src/main/java/com/github/yamill/orientation/util/OrientationUtil.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.yamill.orientation.util
+
+import android.content.res.Configuration
+
+class OrientationUtil {
```

Comment:
nit: If this file is only for static methods, you can define it like this without a companion object.

```
object OrientationUtils {
  fun getOrientationString(orientation: Int) =
            when (orientation) {
                Configuration.ORIENTATION_LANDSCAPE ->
                    "LANDSCAPE"
                Configuration.ORIENTATION_PORTRAIT ->
                    "PORTRAIT"
                Configuration.ORIENTATION_UNDEFINED ->
                    "UNKNOWN"
                else ->
                    null
            }
}
```